.github/workflows/callstack-reviewer.yml

```diff
@@ -0,0 +1,29 @@
+name: Callstack.ai PR Review
+
+on:
+  workflow_dispatch:
+    inputs:
+      config:
+        type: string
+        description: "config for reviewer"
+        required: true
+      head:
+        type: string
+        description: "head commit sha"
+        required: true
+      base:
```

Comment:
:zap: **Suggestion** 
Consider making the 'base' input required to ensure consistency and avoid potential errors during the workflow execution. If 'base' is optional, ensure that the workflow can handle cases where it is not provided.